src/main/java/com/aca/exam/exam3/entity/UserPlateNumber.java

```diff
@@ -0,0 +1,75 @@
+package com.aca.exam.exam3.entity;
+
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "USER_PLATE_NUMBERS")
+public class UserPlateNumber {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_PLATE_NUMBERS_ID_SEQUENCE")
+    @SequenceGenerator(name = "USER_PLATE_NUMBERS_ID_SEQUENCE")
+    private Long id;
+
+    @Column(name = "plate_number")
+    private String plateNumber;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id",
+            foreignKey = @ForeignKey(name = "USER_PLATE_NUMBERS_USER_ID_ID"),
+            referencedColumnName = "id")
+    private User user;
```

Comment:
there can be numbers that do not belong to anyone. in that case the user_id will be null which is not desirable

Reply:
the requirement was to generate all numbers then allow a user to take a number